### ISSUE TITLE
`dropLimit` can be circumvented in certain circumstances

### ISSUE THREAD

**Issue by robartsd** [ArnoldSmith86/virtualtabletop#183]
Under certain circumstances a holder's `dropLimit` property can be circumvented.

By design, we allow the `dropLimit` to be exceeded by automation buttons. A game designer may want to prevent players from adding items beyond a limit but use automation to exceed the limit in some circumstances. In designing the feature, we decided that a user should always be able to return an item to the holder they took the item out of (while still dragging, not after drop). The item being dragged ignores the `dropLimit` of the holder that it was removed from.

Current situation:
Holder is at `dropLimit` so will not allow users to add items. User 1 drags item out of holder. While User 1 is dragging, User 2 adds an item to the holder. User 1 ends the dragging by putting the item back into the holder. The holder is now over the limit.

What we'd like to have happen:
Holder is at `dropLimit` so will not allow users to add items. User 1 drags item out of holder. While User 1 is dragging, User 2 tries to add an item to the holder, but is blocked until User 1's drag ends outside of the holder.

We decided it was not worth delaying the feature to solve this problem.

**Comment by RaphaelAlvez** [comment]
yeah having problem with this right now. drop limit should be respected by move
